data/fileio/CSVFileWriter.h

```diff
@@ -47,6 +59,7 @@ class CSVFileWriter : public QObject
     QString m_error;
     QString m_delimiter;
     DataExportOptions m_options;
+    ProgressReporter *m_reporter;
 };
```

Comment:
Need to update the "old" ctor (the one in the .cpp file) so that it initialises this member (to null presumably). Or use a self-initialising pointer wrapper type.